app/assets/stylesheets/pages/_home.scss

```diff
@@ -11,3 +11,7 @@ $offset: 16%;
     margin-left: $offset;
   }
 }
+
+.i18n-trans-link {
```

Comment:
instead of adding new CSS, can you just add a `ml1` class to the div in the `additional_markup` method?